workflows/edit_data_packages/set_dataone_nodes.Rmd

```diff
@@ -1,10 +1,18 @@
 ## Set DataONE nodes 
-Tell R which repo you want to work with by setting the coordinating node (CN) and member node (MN) using the appropriate code below.
+
+DataONE is a network of data repositories that is structured with coordinating nodes (CN) and member nodes (MN). The network tree looks something like this:
+
+![](../images/dataone-nodes-tree.png)
```

Comment:
@jeanetteclark there are two images folders in the repo, and I think you meant the other one (without `../`) #55

Reply:
@jeanetteclark hmm let me check again.  It doesn't make sense.